WordPress/src/main/java/org/wordpress/android/push/GCMMessageService.java

```diff
@@ -359,34 +359,13 @@ private void buildAndShowNotificationFromNoteData(Context context, Bundle data)
             if (!buildNoteObjectFromPNPayloadAndSaveIt(data)) {
                 // PN payload doesn't have the note or there was an error.
                 // Retrieve the Note obj by calling the REST API
-
-                WordPress.getRestClientUtilsV1_1().getNotification(
-                        wpcomNoteID,
+                NotificationsActions.downloadNoteAndUpdateDB(wpcomNoteID,
```

Comment:
We should probably standardize if we use `noteId` or `noteID` casing, it's quite mixed throughout the code. But that's for another PR :)